ui/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": ["standard", "standard-react"]
+}
```

Comment:
In the RHMAP we had as default use the rules of ngui. Is just this rule enough? Should we apply the same rules here?

Reply:
@camilamacedo86 maybe/ maybe not. The conventions the js community favour may have moved on, but also ngui was backbone, and we will need to include react-specific eslint setting in our config

Reply:
It is fine. The idea of this comment is just for we check: Is just this rule enough for this project? Have someone standard that we should follow here? What rules has in the others aerogear projects?